src/protections/BasicFlooding.ts

```diff
@@ -30,12 +30,11 @@ export class BasicFlooding implements IProtection {
     private lastEvents: { [roomId: string]: { [userId: string]: { originServerTs: number, eventId: string }[] } } = {};
     private recentlyBanned: string[] = [];
 
-    maxPerMinute = new NumberProtectionSetting(DEFAULT_MAX_PER_MINUTE);
-    settings = {};
+    settings = {
+        maxPerMinute: new NumberProtectionSetting(DEFAULT_MAX_PER_MINUTE)
```

Comment:
Hey, did the linter actually catch an error?

If so :v: .

Reply:
yep!